tests/store/api/test_store_api_status.py

```diff
@@ -204,7 +204,7 @@ def test_multiple_analyses(analysis_store, helpers):
 
     # GIVEN an analysis that is not delivery reported but there exists a newer analysis
     timestamp = datetime.now()
-    family = helpers.add_family(analysis_store)
+    family = helpers.add_case(analysis_store)
```

Comment:
```suggestion
    case = helpers.add_case(analysis_store)
```

Reply:
I did a refactor on the method name, I guess that is why it looks like this. I'll fix it

Reply:
fixed